webui/tests/tests/StepImplementation.ts

```diff
@@ -136,6 +140,29 @@ export default class StepImplementation {
     }
   }
 
+  /* Browser Utility Steps */
+
+  @Step('Should not have element <selector> present')
+  public async hasNoElement(selector: string) {
+    const exists = await t.$(selector).exists(undefined, 2000);
+    assert.ok(!exists);
+  }
+
+  @Step('Should have element <selector> present')
+  public async hasElement(selector: string) {
+    await t.$(selector).exists();
+  }
+
+  @Step('Switch to mobile view')
+  public async emulateDevice() {
```

Comment:
**nit:** name it `setMobileViewport`?

Reply:
aah yeah I was originally going with a parameterized step. I'll update